src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -108,4 +108,15 @@ storiesOf('JsonSchemaViewer', module)
         onGoToRef={action('onGoToRef')}
       />
     </div>
+  ))
+  .add('with mask controls', () => (
+    <JsonSchemaViewer
+      maskControlsHandler={() => ['properties/name']}
+      name={text('name', 'my schema')}
+      schema={schema as JSONSchema4}
```

Comment:
To get the grasp of allOf merging we have, you can use the following schema
```suggestion
      schema={allOfSchema as JSONSchema4}
```